File: Web/Akanksha0401_AkankshaMishra_2024cs1126_2/gettingstarted/printallprimestillN.cpp

#include <iostream>
#include <cmath>
using namespace std;
int main(int argc, char **argv){
    int low, high,count;
    cin >> low >> high;
    
    //write your code here
    for(int i=low;i<=high;i++)
     {
         count=2;
         for(int j=2;j<=sqrt(i);j++)
        {
            if(i%j==0)
            count++;
        }
        if(count==2)
        {cout<<i;
        cout<<"\n";
        }
     }
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/PrintAllPrimesTillN.java
import java.util.*;

public class Main{
    public static void main(String[] args) {
        // write your code here
        Scanner sc=new Scanner(System.in);
        int low=sc.nextInt();
        int high=sc.nextInt();
        for(int x=low;x<=high;x++){
            int c=0;
            for(int y=2;y<=Math.sqrt(x);y++){
                if(x%y==0){
                    c++;
                }
             } 
             if(c==0){
                 System.out.println(x);
             }
        }
    }
}